scylla/datadog_checks/scylla/data/conf.yaml.example

```diff
@@ -0,0 +1,152 @@
+    ## <PREFIX> for exposed Prometheus metrics.
+    #
+    # prometheus_metrics_prefix: <PREFIX>_
+
+    ## @param health_service_check - boolean - optional - default: true
+    ## Send a service check reporting about the health of the prometheus endpoint
+    ## It's named <NAMESPACE>.prometheus.health
+    #
+    # health_service_check: true
+
+    ## @param label_to_hostname - string - optional
+    ## Override the hostname with the value of one label.
+    #
+    # label_to_hostname: <LABEL>
+
+    ## @param label_joins - object - optional
+    ## The label join allows to target a metric and retrieve it's label via a 1:1 mapping
+    #
+    # label_joins:
+    #   target_metric:
+    #     label_to_match: <MATCHED_LABEL>
+    #     labels_to_get:
+    #       - <EXTRA_LABEL_1>
+    #       - <EXTRA_LABEL_2>
+
+    ## @param labels_mapper - list of key:value elements - optional
+    ## The label mapper allows you to rename labels.
+    ## Format is <LABEL_TO_RENAME>: <NEW_LABEL_NAME>
+    #
+    # labels_mapper:
+    #   flavor: origin
+
+    ## @param type_overrides - list of key:value elements - optional
+    ## Type override allows you to override a type in the prometheus payload
+    ## or type an untyped metrics (they're ignored by default)
+    ## Supported <METRIC_TYPE> are `gauge`, `counter`, `histogram`, `summary`
+    #
+    # type_overrides:
+    #   <METRIC_NAME>: <METRIC_TYPE>
+
+    ## @param send_histograms_buckets - boolean - optional - default: true
+    ## Set send_histograms_buckets to true to send the histograms bucket.
+    #
+    # send_histograms_buckets: true
+
+    ## @param send_distribution_buckets - boolean - optional - default: false
+    ## Set `send_distribution_buckets` to `true` to send histograms as Datadog distribution metrics.
+    ##
+    ## Learn more about distribution metrics: https://docs.datadoghq.com/developers/metrics/distributions/
+    #
+    # send_distribution_buckets: false
+
+    ## @param send_monotonic_counter - boolean - optional - default: true
+    ## Set send_monotonic_counter to true to send counters as monotonic counter.
+    #
+    # send_monotonic_counter: true
+
+    ## @param send_distribution_counts_as_monotonic - boolean - optional - default: false
+    ## Set send_distribution_counts_as_monotonic to true to send histograms & summary
+    ## counters as monotonic counters (instead of gauges).
+    #
+    # send_distribution_counts_as_monotonic: false
+
+    ## @param exclude_labels - list of strings - optional
+    ## List of label to be excluded
+    #
+    # exclude_labels:
+    #   - timestamp
+
+    ## @param ssl_cert - string - optional
+    ## If your prometheus endpoint is secured, enter the path to the certificate and
+    ## you should specify the private key in ssl_private_key parameter
+    ## or it can be the path to a file containing both the certificate & the private key
+    #
+    # ssl_cert: "<CERT_PATH>"
```

Comment:
Instead of the ssl config options, we can now just use the `tls` ones because the openmetrics base class supports the HTTP wrapper.

Reply:
They can be found in other integration conf.yaml that use the wrapper. But here are the few that correspond to the 3 in this config.

```
    ## @param tls_verify - boolean - optional - default: true
    ## Instructs the check to validate the TLS certificate of services.
    #
    # tls_verify: true

    ## @param tls_ignore_warning - boolean - optional - default: false
    ## If `tls_verify` is disabled, security warnings are logged by the check.
    ## Disable those by setting `tls_ignore_warning` to true.
    #
    # tls_ignore_warning: false

    ## @param tls_cert - string - optional
    ## The path to a single file in PEM format containing a certificate as well as any
    ## number of CA certificates needed to establish the certificate's authenticity for
    ## use when connecting to services. It may also contain an unencrypted private key to use.
    #
    # tls_cert: <CERT_PATH>

    ## @param tls_private_key - string - optional
    ## The unencrypted private key to use for `tls_cert` when connecting to services. This is
    ## required if `tls_cert` is set and it does not already contain a private key.
    #
    # tls_private_key: <PRIVATE_KEY_PATH>

    ## @param tls_ca_cert - string - optional
    ## The path to a file of concatenated CA certificates in PEM format or a directory
    ## containing several CA certificates in PEM format. If a directory, the directory
    ## must have been processed using the c_rehash utility supplied with OpenSSL. See:
    ## https://www.openssl.org/docs/manmaster/man3/SSL_CTX_load_verify_locations.html
    #
    # tls_ca_cert: <CA_CERT_PATH>
```